compiler/AST/build.cpp

```diff
@@ -1364,7 +1364,7 @@ BlockStmt* buildCoforallLoopStmt(Expr* indices,
     //
     VarSymbol* coforallCount = newTemp("_coforallCount");
     BlockStmt* block = ForLoop::buildForLoop(indices, iterator, body, true, zippered);
-    block->insertAtHead(new CallExpr(PRIM_MOVE, coforallCount, new CallExpr("_endCountAlloc")));
+    block->insertAtHead(new CallExpr(PRIM_MOVE, coforallCount, new CallExpr("_endCountAlloc", gFalse)));
```

Comment:
I think it would really help if you put a comment next to gTrue/gFalse in these AST building calls to indicate that the 2nd argument either forces a local type or allows network atomics.